source/backend/api/Services/ProjectService.cs

```diff
@@ -57,6 +57,18 @@ public class ProjectService : BaseService, IProjectService
             _entityNoteRepository = entityNoteRepository;
         }
 
+        public IList<PimsProject> GetProjects()
+        {
+            _logger.LogInformation("Retrieving all projects");
+            _user.ThrowIfNotAuthorized(Permissions.ProjectView);
+
+            // Limit search results to user's assigned region(s), but always include "Cannot determine" region
+            var pimsUser = _userRepository.GetUserInfoByKeycloakUserId(_user.GetUserKey());
```

Comment:
does this require any claims the average user may not have? I don't remember.

Reply:
no need for this api anymore now that is using the lookup